packages/lab/src/stepped-tracker/SteppedTracker.tsx

```diff
@@ -34,6 +34,10 @@ export interface SteppedTrackerProps extends ComponentPropsWithoutRef<"ul"> {
    * The orientation of the SteppedTracker. Defaults to `horizontal`
    */
   orientation?: "horizontal" | "vertical";
+  /**
+   * Render in visually compact mode.
+   */
+  compact?: boolean;
```

Comment:
just to respond to your discussion points, it doesn't seem ideal to have a compact prop that only works with one orientation
if it's not going to be a separate component then I think 'horizontal-compact' orientation prop makes sense